src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -69,6 +75,11 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 
 	}
 
+	private contentFormatter(dataValue: string, row: number | undefined, cell: any | undefined, value: any, columnDef: any | undefined, dataContext: any | undefined): string {
+		let cellClasses = 'grid-cell-value-container';
+		return `<div class="${cellClasses}">${dataContext[dataValue]}</div>`;
```

Comment:
What's dataContext here?

And it looks like this is also going to directly embedding content from the extension without escaping it? If so that's another thing we need to look at closer so I'd also suggest removing this for now and then following up with a separate PR so we can look into supporting what you need in a safe manner.